individual-files/ServerThread.java

```diff
@@ -0,0 +1,49 @@
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.Socket;
+import java.util.ArrayList;
```

Comment:
Extraneous (E) - lines 5 and 6 are especially unnecessary as neither lists nor dynamically allocated lists are utilized in this code.